src/i2p.cpp

```diff
@@ -370,12 +372,12 @@ void Session::CreateIfNotCreatedAlready()
               m_my_addr.ToString());
 }
 
-Sock Session::StreamAccept()
+std::unique_ptr<Sock> Session::StreamAccept()
 {
-    Sock sock = Hello();
+    auto sock = Hello();
```

Comment:
5ac2bc1 Verified this line and line 352 compile when replaced with explicit typing and braced initialization `std::unique_ptr<Sock> sock{Hello()};` (which would save people time in looking up the type)